src/components/PurchaseOrder/util.js

```diff
@@ -124,3 +124,25 @@ export const getPoFieldsLabelMap = () => {
     'closeReason.note': 'ui-orders.orderSummary.closingNote',
   };
 };
+
+export const getPrefixSuffixOptions = (
+  records,
+  selectedValue,
+  intl,
+) =>
+  records
+    .filter(({ name, deprecated }) => !deprecated || name === selectedValue)
+    .map(({ name, deprecated }) => {
+      const deprecatedText = intl.formatMessage(
```

Comment:
ideally to not call formatting for non-deprecated records (not big optimization from performance, but still)